dashboard/app/models/pd/application/teacher1920_application.rb

```diff
@@ -157,11 +157,6 @@ def principal_approval_state
       nil
     end
 
-    def formatted_teacher_email
```

Comment:
I renamed this and moved it to application base so facilitator apps can use it too